yaml/landuse.yaml

```diff
@@ -458,6 +458,13 @@ filters:
       kind: airfield
       kind_detail: military
       tier: 3
+  - filter: {military: naval_base}
+    min_zoom: *tier3_min_zoom
+    output:
+      <<: *output_properties
+      kind: naval_base
+      kind_detail: military
```

Comment:
Why is the `kind_detail` set here?  Are there naval bases that aren't military related?

Reply:
Ah, sorry, I think that was copy-pasta from the above `airfield`, where it makes sense to say it's a military thing. Civilian airfields exist, but civilian naval bases, as you rightly point out, not so much. Fixed in 96110df.